Podcast/PlayButton.swift

```diff
@@ -10,8 +10,12 @@ import UIKit
 
 class PlayButton: UIButton {
     
-    init() {
-        super.init(frame: .zero)
+    convenience init() {
```

Comment:
what is the point of this when all it does is just do the same thing as initializing with UIButton()